doc/howtos/rdtraining/16_guidelines_pr.rst

```diff
@@ -0,0 +1,154 @@
+    $ git checkout master #Two new files will be downloaded README.md and .gitignore
+
+Note that if your work is already in a :ref:`Git and GitHub configured folder <howto/rdtraining/02_setup/development_repository>`
+the previous steps are not needed. This is going to be the case when you will start working in the
+'odoo' or 'enterprise' folders.
+
+
+Branch, Commit & Push
+---------------------
+
+Before creating the PR, a new working branch has to be created and used to commit the code. Afterwards
+the branch will be pushed to the development repository.
+
+Create your working branch:
+
+.. code-block:: console
+
+    $ git checkout -b master-my_first_branch-xyz
+
+Your branch name must follow the following name strucutre : <targetVersion>-<feature>-<trigram>
+
+Example: The branch master-sale-fixes-abc on odoo-dev/odoo is a branch containing fixes for the
+sales app in the odoo/odoo repository, to be deployed in master and done by ABC.
+
+Commit your code:
+
+
+.. code-block:: console
+
+    $ git add .
+    $ git commit
+
+
+**Everyone is reading your commit messages!**
+
+The commit message is very important, follow the :ref:`Developer guidelines <reference/guidelines/git>`.
+
+
+Push your new branch to your developement repository:
+
+.. code-block:: console
+
+    $ git push -u master-my_first_branch-xyz
+
+.. warning:: NEVER use `git push --force` (or `git push -f`) in a branch that does not belong to you
+             nor in a stable branch. Never, never, never!
+
+.. tip:: Remember that you can use `git status` at any step to know the current status of your branch
+.. tip:: You can create git alias to shorten some of theses commands. Ask your colleagues around you
+         if they use git aliases and get inspired by their work.
+
+Create your first PR
+--------------------
+
+After pushing your branch to your development repository, the output will look like this:
+
+.. code-block:: console
+
+    Enumerating objects: 24, done.
+    Counting objects: 100% (24/24), done.
+    Compressing objects: 100% (20/20), done.
+    Writing objects: 100% (23/23), 6.67 KiB | 525.00 KiB/s, done.
+    Total 23 (delta 1), reused 0 (delta 0)
+    remote: Resolving deltas: 100% (1/1), done.
+    remote:
+    remote: Create a pull request for 'master-my_first_branch-xyz' on GitHub by visiting:
+    remote:      https://github.com/xyz-odoo/technical-training-sandbox/pull/new/master-my_first_branch-xyz
+    remote:
+    To github.com:xyz-odoo/technical-training-sandbox.git
+    * [new branch]      master-my_first_branch-xyz -> master-my_first_branch-xyz
+    Branch 'master-my_first_branch-xyz' set up to track remote branch 'master-my_first_branch-xyz' from 'training-dev'.
+
+There are two ways to create the PR:
+
+- Click on the link displayed in the output of the push command.
+- Open a browser in your development repository `https://github.com/xyz-odoo/technical-training-sandbox`.
+  A button will suggest you to create a new pull request.
```

Comment:
```suggestion
  A button will suggest that you create a new pull request.
```

or "the creation of"

Reply:
Also maybe the technical-training-sandbox should be set up such that github always picks the wrong branch? Since github targets the default branch by default but since for Odoo it's the current stable and most devs don't work on it (working either on master or a recent saas for most and on older branches for bugfix) GH's pick almost never works (and usually takes 5mn to realise 27648 commits is a bit much because it's trying to merge master into 13.0).